### PR TITLE
add aliyun oss example

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8739
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-storages.md [hunk 1]
```diff
@@ -156,6 +157,43 @@ If you have specified URL parameters and command-line parameters at the same tim
 | `--s3.acl` | The canned ACL of the upload object. For example, `private` and `authenticated-read`. |
 | `--s3.provider` | The type of the S3-compatible service. The supported types are `aws`, `alibaba`, `ceph`, `netease` and `other`. |
 
+To export data to non-AWS S3 cloud storage, you should specify the cloud provider and whether to use `virtual-hosted style`. The following exemplifies storage to Alibaba Cloud OSS:
```
Comment (TomShawn) [inline br/backup-and-restore-storages.md:160]:
```suggestion
To export data to non-AWS S3 cloud storage, specify the cloud provider and whether to use `virtual-hosted style`. In the following examples, data is exported to the Alibaba Cloud OSS storage:
```

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -156,6 +157,43 @@ If you have specified URL parameters and command-line parameters at the same tim
...
+    ./br backup full --pd "127.0.0.1:2379" \
+        --storage "s3://my-bucket/full/" \
+        --s3.endpoint="http://oss-cn-hangzhou-internal.aliyuncs.com" \
+        --s3.provider="alibaba" \
+        --send-credentials-to-tikv=true \
+        --ratelimit 128 \
+        --log-file backuptable.log
+    ```
+
+* Export data from TiDB Lightning to Alibaba Cloud OSS:
```
Comment (TomShawn) [inline br/backup-and-restore-storages.md:188]:
```suggestion
* Export data to Alibaba Cloud OSS using TiDB Lightning. You need to specify the following content in the YAML-formatted configuration file:
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: br/backup-and-restore-storages.md [hunk 3]
```diff
@@ -78,6 +78,7 @@ Cloud storages such as S3, GCS and Azblob sometimes require additional configura
 | `region` | Service Region for Amazon S3 (default to `us-east-1`) |
 | `use-accelerate-endpoint` | Whether to use the accelerate endpoint on Amazon S3 (default to `false`) |
 | `endpoint` | URL of custom endpoint for S3-compatible services (for example, `https://s3.example.com/`) |
+| `force-path-style` | Use `path-style`, instead of `virtual-hosted style` (default to `true`) |
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply br/backup-and-restore-storages.md]: Fixed.

**Pull request merged**